src/goods.ts

```diff
@@ -53,16 +53,19 @@ export async function fetch(url: RequestInfo, init?: RequestInit) {
 }
 
 // A console.log() alternative which can take ProcessOutput.
-export function echo(pieces: TemplateStringsArray, ...args: any[]) {
+export function echo(
+  pieces: TemplateStringsArray | ProcessOutput | string,
```

Comment:
This is not right, as argument can be `any`. Also arity should be one. 

This better to fix with a signature override.